jekyll/_cci2/arm-resources.md

```diff
@@ -18,7 +18,8 @@ resource on CircleCI. Arm resources are available on cloud and server 3.x.
 **Note:** Docker images built on M1 Macs, are by default, not compatible with the CircleCI standard platform.
 The Spin Up Environment job will give you a green tic, but you will see the following message in the response:
   
-```WARNING: docker image ghcr.io/{your_username}/runner-test:latest targets wrong architecture (found arm64 but need [amd64 i386 386])```
+```WARNING: docker image ghcr.io/{your_username}/runner-test:latest targets wrong architecture (found arm64 but need [amd64 i386 386])
```

Comment:
@thomasr8 can you double check this is proper syntax? There seems to be a space and a line break from the end of the code to the back ticks, which would break the formatting. The line above that was deleted actually seems correct (I think you can use one back tick instead of three if you want).

Reply:
fixed